crates/aptos/src/account/multisig_account.rs

```diff
@@ -0,0 +1,253 @@
+                WriteSetChange::WriteResource(WriteResource { address, data, .. }) => {
+                    if data.typ.name.as_str() == "Account"
+                        && *address.inner().to_hex() != *txn.request.sender.inner().to_hex()
+                    {
+                        Some(MultisigAccount {
+                            multisig_address: *address.inner(),
+                        })
+                    } else {
+                        None
+                    }
+                },
+                _ => None,
+            });
+        }
+
+        summary
+    }
+}
+
+#[async_trait]
+impl CliCommand<CreateSummary> for Create {
+    fn command_name(&self) -> &'static str {
+        "CreateMultisig"
+    }
+
+    async fn execute(self) -> CliTypedResult<CreateSummary> {
+        self.txn_options
+            .submit_transaction(aptos_stdlib::multisig_account_create_with_owners(
+                self.additional_owners,
+                self.num_signatures_required,
+                // TODO: Support passing in custom metadata.
+                vec![],
+                vec![],
+            ))
+            .await
+            .map(CreateSummary::from)
+    }
+}
+
+/// Propose a new multisig transaction.
+///
+/// As one of the owners of the multisig, propose a new transaction. This also implicitly approves
+/// the created transaction so it has one approval initially. In order for the transaction to be
+/// executed, it needs as many approvals as the number of signatures required.
+#[derive(Debug, Parser)]
+pub struct CreateTransaction {
+    #[clap(flatten)]
+    pub(crate) multisig_account: MultisigAccount,
+    #[clap(flatten)]
+    pub(crate) txn_options: TransactionOptions,
+    #[clap(flatten)]
+    pub(crate) entry_function_args: EntryFunctionArguments,
+}
+
+#[async_trait]
+impl CliCommand<TransactionSummary> for CreateTransaction {
+    fn command_name(&self) -> &'static str {
+        "CreateTransactionMultisig"
+    }
+
+    async fn execute(self) -> CliTypedResult<TransactionSummary> {
+        let payload = MultisigTransactionPayload::EntryFunction(
+            self.entry_function_args.create_entry_function_payload()?,
+        );
+        self.txn_options
+            .submit_transaction(aptos_stdlib::multisig_account_create_transaction(
+                self.multisig_account.multisig_address,
+                to_bytes(&payload)?,
+            ))
+            .await
+            .map(|inner| inner.into())
+    }
+}
+
+/// Approve a multisig transaction.
+///
+/// As one of the owners of the multisig, approve a transaction proposed for the multisig.
+/// With enough approvals (as many as the number of signatures required), the transaction can be
+/// executed (See Execute).
+#[derive(Debug, Parser)]
+pub struct Approve {
+    #[clap(flatten)]
+    pub(crate) multisig_account: MultisigAccount,
+    #[clap(long)]
+    pub(crate) sequence_number: u64,
```

Comment:
We should add a doc comment saying this is the `multisig` sequence number, rather than any of the other ones

Reply:
Added